aws-cli/plan.sh

```diff
@@ -9,37 +9,8 @@ pkg_description="The AWS Command Line Interface (CLI) is a unified tool to \
   through scripts."
 pkg_upstream_url=https://aws.amazon.com/cli/
 pkg_source=nosuchfile.tgz
```

Comment:
This really through me off when I was just trying to make `pip` source only plans. We might want to make this `$pkg_source=pip` or possibly what would be better is if `$pkg_scaffolding_python_pip_name` is fleshed out then we can safely assume `$pkg_source=pip`.